WRITING_DOCS.md

```diff
@@ -9,8 +9,6 @@
   - [Divider](#divider)
   - [Grid](#grid)
   - [Mini-Card](#mini-card)
-  - [Dark-Mode](#dark-mode)
```

Comment:
The estimated reading times changed between some pages.
![image](https://github.com/icerpc/icerpc-docs/assets/25963603/ba49c446-5780-4858-a734-e6d41e8bf38c)
But looking at the pages, it doesn't look like any content actually changed.

They're pretty similar, but still, it's just weird that it happened since nothing _seems_ to of changed.

Reply:
The old 2 minute time seems more accurate.
Obviously a sample size of 1 is useless, but it took me 1:24 to read the page.

Reply:
The logic had to change slightly to allow us to SSR the reading time. It seems to be pretty close still. In fact the new version should be a bit more accurate. We now compute it by looking at the markdown content instead of the rendered html.

Reply:
Yeah, most pages are still the same, and the ones that were different were all within a minute.
It just seemed weird so I though I'd let you know!